packages/pangea-sdk/PangeaCyber.Net/Utils/Utils.cs

```diff
@@ -29,6 +28,28 @@ public static string Base64ToString(string input)
             return Encoding.UTF8.GetString(bytes);
         }
 
+        /// <summary>
+        /// Base 64 decoding with an URL and filename safe alphabet. Also known as "base64url".
+        /// Supports decoding without padding.
+        /// </summary>
+        /// <remarks><see href="https://datatracker.ietf.org/doc/html/rfc4648#section-5" /></remarks>
+        /// <param name="base64">Base 64 encoded string.</param>
+        /// <returns>Base 64 decoded bytes.</returns>
+        public static byte[] Base64UrlDecode(string base64)
```

Comment:
Why not using a package? It seems that it's already resolved... https://www.geeksforgeeks.org/url-encoding-decoding-using-base64-in-java/

Reply:
Alright, removed this. Your link is for Java so instead I brought in Microsoft.IdentityModel.Tokens for its `Base64UrlEncoder`.

Reply:
Yes, my bad, sorry